pkgs/development/python-modules/lizard/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "lizard";
+  version = "1.17.10";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1p2s0wx1l75dsljiyvars7xlljzs1mipm8jaby5y4jvjck6qmmv2";
+  };
+
+  pythonImportsCheck = [
+    "lizard"
+  ];
+
+  doCheck = false;
```

Comment:
The repo seems to contain [tests](https://github.com/terryyin/lizard/tree/master/test).

Reply:
good call - another one where i spaced on pulling from github instead of pypi 🤦‍♂️